### ISSUE TITLE
Description and Review Tabs don't work

### ISSUE THREAD

**Issue by navjotjsingh** [jameskoster/highwind#72]
Description and Review Tabs don't work as they should on the product pages. This is how they look right now with the default setup.

![](http://i.imgur.com/f7DkvfG.png)

Am using WordPress 3.8.1 and Woocommerce 2.1.6 with latest version of Highwind

**Comment by jameskoster** [comment]
Working fine for me. Check for javascript errors/plugin conflicts please.

**Comment by navjotjsingh** [comment]
This exact same thing happened on two fresh installs - localhost and live blog. No other plugin was installed. I didn't even change any default setting on the plugin or the theme. I installed a fresh blog. Installed the theme and activated it. Installed Woocommerce and activated it. And just added a test product and what I get is what I posted in the screenshot above. Where to look for javascript issue in this case?

Only thing I get in the javascript console is this error:

`event.returnValue is deprecated. Please use the standard event.preventDefault() instead.`

**Comment by jameskoster** [comment]
It's almost certainly a javascript issue. The tabs aren't working, the script that replaces the rating select with stars isn't working either. Can I see a URL?

**Comment by navjotjsingh** [comment]
Here http://labs.nspeaks.com/?product=test-product Its a test blog I have setup. Nothing has been configured at all. No other plugins installed.

**Comment by jameskoster** [comment]
Yeah that's weird. Not really sure what to suggest. I just viewed the source of that page and none of the WooCommerce js is being loaded. This isn't an issue with the theme though, I suggest posting on the .org support forums.

**Comment by navjotjsingh** [comment]
Actually if I switch the theme to let's say Twenty Fourteen, the tabs start working fine. Check the url again http://labs.nspeaks.com/?product=test-product

Am not sure whether its a Woocommerce specific problem.

**Comment by navjotjsingh** [comment]
I guess I found the problem.

In the files includes/integrations/woocommerce

functions.php and template.php, you have used

```
add_action( 'woocommerce_after_single_product', 'woocommerce_upsell_display', 15 );
```

and 

```
function woocommerce_upsell_display() {
    woocommerce_upsell_display( -1, 3 );
}
```

Changing it to

```
add_action( 'woocommerce_after_single_product', 'highwind_woocommerce_upsell_display', 15 );
```

and 

```
function highwind_woocommerce_upsell_display() {
    woocommerce_upsell_display( -1, 3 );
}
```

solves the problem. Not only this, this change also fixes #73 as well.

**Comment by jameskoster** [comment]
#winning
